packages/loopring_v3/contracts/iface/ExchangeData.sol

```diff
@@ -72,15 +58,23 @@ library ExchangeData
         uint8 previousMakerFeeBips;
     }
 
+    // General auxiliary data for each conditional transaction
+    struct AuxiliaryData
+    {
+        uint txIndex;
+        bytes data;
+    }
+
     // This is the (virtual) block an operator needs to submit onchain to maintain the
     // per-exchange (virtual) blockchain.
     struct Block
     {
-        ExchangeData.BlockType blockType;
+        uint8                  blockType;
```

Comment:
I left it in because we may still want to have dedicated circuits for some transaction types at some point. Probably not, but doesn't really hurt to keep for now I'd think.